WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductFromImage/AddProductFromImageEligibilityChecker.swift

```diff
@@ -34,7 +38,10 @@ final class AddProductFromImageEligibilityChecker: AddProductFromImageEligibilit
             return false
         }
 
-        // TODO: 10180 - A/B experiment check
-        return featureFlagService.isFeatureFlagEnabled(.addProductFromImage)
+        guard featureFlagService.isFeatureFlagEnabled(.addProductFromImage) else {
+            return false
+        }
```

Comment:
From what I understand, using the local feature flag was a temporary solution before the A/B test is available. We should remove this in favor of the A/B test check.

Reply:
I thought it would be convenient to have a feature flag as an additional switch. I will remove it and update the release notes.

Reply:
[Removed feature flag](https://github.com/woocommerce/woocommerce-ios/pull/10253/commits/b82415fa023af931ba9e53cf640a5e20df8d0905) and [added release notes](https://github.com/woocommerce/woocommerce-ios/pull/10253/commits/54062afeb822da5e1fc2ee2680548144cc6e2d41).